### ISSUE TITLE
can't get the list of tasks from grafana dashboard

### ISSUE THREAD

**Issue by irom77** [raintank/snap-app#2]
I have snap DS http://10.73.21.205:8181 configured in grafana 3.0 (running on separate host than DS) but can't get the list of tasks from grafana dashboard GUI when I try Graph->Task name->select task.  From grafana host I can reach DS and list tasks without any problem, see below

```
$ curl -L http://10.73.21.205:8181/v1/tasks
{
  "meta": {
    "code": 200,
    "message": "Scheduled tasks retrieved",
    "type": "scheduled_task_list_returned",
    "version": 1
  },
  "body": {
    "ScheduledTasks": [
      {
        "id": "13b9f7d5-614c-41f1-a65b-526d517843b1",
        "name": "Task-13b9f7d5-614c-41f1-a65b-526d517843b1",
        "deadline": "5s",
        "creation_timestamp": 1468610189,
        "last_run_timestamp": 1468611289,
        "hit_count": 220,
        "task_state": "Running",
        "href": "http://10.73.21.205:8181/v1/tasks/13b9f7d5-614c-41f1-a65b-526d517843b1"
      }
    ]
  }
```

**Comment by ommoreno** [comment]
I am also having this issue, but with snapd running on the same host as grafana. I can add snap as datasource, but when it comes to adding a task and its metrics to a dashboard graph, there are no drop down options.

**Comment by theluke55** [comment]
i'm also getting this behavior like @ommoreno 

i have snapped running on same host as grafana, but when i click on metric nothing shows up

following this tutorial: https://github.com/intelsdi-x/snap-demo-idf/blob/master/using-snap/explore-snap.md

**Comment by theluke55** [comment]
i was able to fix this by running grafana v3.x.x

**Comment by ommoreno** [comment]
I'm running Grafana v3.1.1 (commit: a4d2708) and I still have this issue.

**Comment by theluke55** [comment]
@ommoreno for my data source i used this

url: http://localhost:8181
access: proxy

also are you trying to do this behind a proxy.

I'm guessing your setup is snapd and grafana on remote server, and accessing with local laptop or machine?

**Comment by ommoreno** [comment]
@theluke55 Yes, I am also behind a proxy accessing remotely on my local machine.

It looks like running grafana v4.1.1 and snap v1.0.0  fixes this issue.

Now I'm running into another issue where nothing is graphed even after selecting or creating a task and watching it. I may open another thread for it.
